Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -9,12 +9,10 @@
 import OpenAI
 import SwiftUI
 
-
 /// Displays the content of a `Chat` message in a message bubble
 public struct MessageView: View {
     let chat: Chat
-    let hideSystemMessages: Bool
-    
+    let hideMessagesWithRoles: Set<Chat.Role>
```

Comment:
Please add two empty lines between stored properties and computed properties:
```suggestion
    let hideMessagesWithRoles: Set<Chat.Role>
    
```